src/WebJobs.Script/Rpc/LanguageWorkerProcess.cs

```diff
@@ -35,15 +35,15 @@ internal class LanguageWorkerProcess : ILanguageWorkerProcess, IDisposable
                                        IWorkerProcessFactory processFactory,
                                        IProcessRegistry processRegistry,
                                        ILogger workerProcessLogger,
-                                       ILanguageWorkerConsoleLogSource consoleLogSource)
+                                       ILoggerFactory loggerFactory)
         {
             _runtime = runtime;
             _workerId = workerId;
             _processFactory = processFactory;
             _processRegistry = processRegistry;
             _workerProcessLogger = workerProcessLogger;
-            _consoleLogSource = consoleLogSource;
             _eventManager = eventManager;
+            _consoleLogger = loggerFactory.CreateLogger(LanguageWorkerConstants.FunctionConsoleLogCategoryName);
```

Comment:
We can't do that here -- since we're creating a specific category (for filtering out of the kusto logs) and not `ILogger<T>`, we need to explicitly use the loggerfactory.

Reply:
Just a NIT: `LanguageWorkerConstants.FunctionConsoleLogCategoryName` is a constant and it will be good to create logger in: https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Rpc/LanguageWorkerProcessFactory.cs